main.js

```diff
@@ -187,3 +187,7 @@ function removeClass() {
         }
     }
 }
+
+$(function() {
```

Comment:
This is the first JQueryism that I've introduced. Because the javascript is no longer running synchronously with the page load, we can't run functions that operate on DOM elements until after the DOM has actually been populated. JQuery provides this syntax to specify a function that will be called once the DOM is ready to be used, so we don't need to worry about the details